pkg/collector/python/check.go

```diff
@@ -78,7 +78,7 @@ func (c *PythonCheck) runCheck(commitMetrics bool) error {
 	}
 	defer gstate.unlock()
 
-	log.Debugf("Running python check %s %s", c.ModuleName, c.id)
+	log.Debugf("Running python check %s version %s with ID %s", c.ModuleName, c.version, c.id)
```

Comment:
What do you think about this:
```suggestion
	log.Debugf("Running python check %s (version: '%s', id: '%s')", c.ModuleName, c.version, c.id)
```

Follow-up: can version be nil/empty string? If so, we may need to have a special case for it.

Reply:
Default is https://github.com/DataDog/datadog-agent/blob/3732aadd51804ef2251b4f5ac62cef6788e248f6/pkg/collector/python/loader.go#L160